test/jest/logger.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { Logger, ConsoleLogger } from '../../src/logger';
+
+describe('Logger', () => {
+    let mockConsole: any;
+    let logger: Logger;
+
+    beforeEach(() => {
+        mockConsole = {
+            info: jest.fn(),
+            warn: jest.fn(),
+            error: jest.fn()
+        };
+        (global as any).console = mockConsole;
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should call verbose, warning, and error methods on ConsoleLogger at correct log levels', () => {
+        logger = new Logger({ logLevel: 'verbose' as any });
```

Comment:
Import `LogLevelType` and use it here and below instead of using `as any`


```suggestion
        logger = new Logger({ logLevel:  `LogLevelType.Verbose` });
```